lib/tasks/export.rake

```diff
@@ -60,4 +61,20 @@ namespace :export do
     exporter = CicognaraMarc.new(cico_collection_id: coll, out_dir: output_dir)
     exporter.run
   end
+
+  desc "Export IIIF manifest links to PULFA DAOs"
+  task pulfa: :environment do
+    begin
+      since = ENV["SINCE"] || (Time.zone.today - 7).strftime("%Y-%m-%d")
+      PulfaExporter.new(since_date: since).export
+    rescue PulfaExporter::SvnClient::SvnDirectoryError => e
+      puts e.to_s
+    end
+  end
+
+  desc "Export PDFs to PULFA DAOs"
```

Comment:
Can we add a comment describing the use case here? I think this is for loading PDFs onto the locked-down network machine? It is intended to be run manually as-needed? Or maybe this is documented somewhere else?

Reply:
I'll add a comment, but I also created a ticket for doing that automation work: https://github.com/pulibrary/figgy/issues/3411